<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_EnginePreferences</class>
 <widget class="QWidget" name="GUI_EnginePreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="rb_auto">
     <property name="text">
      <string notr="true">Automatic</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="rb_alsa">
     <property name="text">
      <string notr="true">Alsa</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="combo_alsa_devices"/>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="rb_pulse">
     <property name="text">
      <string notr="true">Pulseaudio</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
